_posts/2024-11-10-5.10.0-released.md

```diff
@@ -0,0 +1,105 @@
+description: >-
+  Luanti 5.10.0 has been released with 
+image: /static/blog/5.10.0/cover.webp
+image_credit: Gefüllte Taubenbrust
+forum_topic: https://forum.luanti.org/viewtopic.php?p=440491
+tags:
+  - release
+---
+
+<h2 class="sr-only">Introduction</h2>
+
+Luanti ([formerly Minetest](https://blog.minetest.net/2024/10/13/Introducing-Our-New-Name/)) 5.10.0 has been released
+just three months after 5.9.1 in accordance with the new release schedule.
+
+Highlights include a **content browser redesign**, support for the modern **glTF** model format, gettext translation
+support, new visual effects, an API for per-player objects, usability improvements, and more.
+
+<!-- more -->
+
+You can download Luanti from [our website](https://www.luanti.org/downloads/). As always, the full changelog can be
+found [here](https://dev.minetest.net/Changelog#5.9.1_.E2.86.92_5.10.0).
+
+### Table of Contents
+
+- [Content Browser Redesign](#content-browser-redesign)
+- [glTF Support](#gltf-support)
+- [gettext Translations](#gettext-translations)
+- [Visual Effects](#visual-effects)
+- [Per-player Objects](#per-player-objects)
+- [More Highlights](#more-highlights)
+- [Conclusion](#conclusion)
+
+## Content Browser Redesign
+
+Luanti's built-in content browser was redesigned to be more user-friendly and appealing, presenting content (mods, games
+or texture packs) in a grid.
+
+{% include figure.html src="/static/blog/5.10.0/content_browser_grid.webp" caption="The new content browser grid" %}
+
+You can now also view the details for a package, similar to the ContentDB web interface:
+
+{% include figure.html src="/static/blog/5.10.0/content_browser_details.webp" caption="The new content details view" %}
+
+## glTF Support
+
+Until now, Luanti developers had to choose between two antiquated file formats for animated 3D models (`.b3d` and `.x`)
+which have poor support in modern software and are difficult to work with.
+
+Luanti now supports the modern [glTF file format](https://www.khronos.org/glTF/), which allows for seamless exports and
+imports with modeling software such as Blender or Blockbench. It also allows developers to easily drop in freely-licensed
+glTF assets from the internet.
+
+[Support](https://github.com/minetest/minetest/blob/5.10.0/doc/lua_api.md#gltf) is still a bit rough around the edges,
+but feature parity with existing model file formats has already been achieved and better support is planned for the
+future.
+
+## gettext Translations
+
+Luanti finally supports the standard [gettext](https://www.gnu.org/software/gettext/) `.po` and `.mo` translation file
+formats! These formats are much easier to manage via external tools such as Weblate, and also provide more options for
+translators (such as correct plural forms). The custom `.tr` translation file format is still supported, but we highly
+recommend migrating to gettext (see [Translations API](https://github.com/minetest/minetest/blob/5.10.0/doc/lua_api.md#translations)).
```

Comment:
I don't think we should highly recommend switching to PO file immediately since it would not work with older clients.

(That said, it would be nice to suggest using PO files for newer mods that use e.g. Weblate).

Reply:
Good point, noted the caveat: https://github.com/minetest/blog/pull/154/commits/9319e9a491ae2621873a363af184c35bb0d1f5cc